tests/scripts/docker/remove.sh

```diff
@@ -1,12 +0,0 @@
-#!/usr/bin/env bash
```

Comment:
How is the cleanup happening now?

Reply:
Since we do not build new images anymore but just use plain `python` images I don't think it is necessary to remove them again afterwards.

Reply:
Got it, makes sense. 

For the cleanup I was more thinking of volumes and network interfaces, to make sure they are removed every time at the end of a build, also if the build crashes.

On the beats side we had several issues in the past where tests were unpredictable because volumes were reused or unused volumes filled up the disk over time. Same for network interfaces, as there is a limit of something like 25 network interfaces max.

I think something like `docker prune ...` or `docker-compose down ...` should do the trick.